docs/general/cloud-events.md

```diff
@@ -286,6 +286,29 @@ The application then may get the latest terms of use through the [Get Terms Of U
 }
 ```
 
+## Consent update events
+
+Fired whenever a user grants or revokes a consent. A whole deletion of a consent triggers a revoked event.
+Applications that are part of any of the users subscriptions receive a notification. The application can then retrieve the latest consense status of the user.
```

Comment:
"Application is part of users subscription" ? Unclear what you mean.
A subscription can have a license of an application.
"users subscription" means I guess  "user's subscription"

Further: get the consent information, such as consented scope, from the Consent API.
Where we need to add the "Consent API" to https://developer.tapio.one/general/available-apis when APIM story is implemented.